docroot/modules/custom/mass_views/mass_views.module

```diff
@@ -229,3 +229,27 @@ function mass_views_do_invalidate($entity): void {
     Cache::invalidateTags($tags);
   }
 }
+
+/**
+ * Implements hook_views_pre_render().
+ */
+function mass_views_views_pre_render(ViewExecutable $view) {
+  if ($view->id() == 'collection_all') {
+    foreach ($view->result as $row) {
+      // Check if the row entity is a media entity
+      if ($row->_entity == NULL && $row->entity_type == 'media') {
+        // Access the fields of the row
```

Comment:
What do we mean by 'access'? Did you mean unset? Would be good to say why we are doing it, and not just what we are doing.

Reply:
Added a more detailed explanation.